docs/readthedocs/source/doc/Nano/Howto/accelerate_pytorch_inference_onnx.md

```diff
@@ -0,0 +1,49 @@
+# Accelerate PyTorch Inference using ONNXRuntime
+
+To accelerate your PyTorch inference pipeline through ONNXRuntime as backend engine, BigDL-Nano provides you with `Trainer.trace(accelerator='onnxruntime')` API. Based on that, the optimization could be implemented by applying very few lines of code changes.
+
+Let us suppose we have a [ResNet-18 model](https://pytorch.org/vision/main/models/generated/torchvision.models.resnet18.html) pretrained on ImageNet dataset, and would like to do the inference on a randomly generated test dataset. The model would be like: 
```

Comment:
Let's take a [ResNet-18 model](https://pytorch.org/vision/main/models/generated/torchvision.models.resnet18.html) pretrained on ImageNet dataset as an example. First we load the model:
```python
from torchvision.models import resnet18
model_ft = resnet18(pretrained=True)
```
Then we set it to the evaluation mode. 
```python
model_ft.eval()
```

Reply:
Have revised :D